vignettes/dm.Rmd

```diff
@@ -51,17 +52,37 @@ fin_dm
 The dm object interrogates the RDBMS for table and column information and, where implemented, primary and foreign keys.
 Currently, primary and foreign keys are only available from Postgres and SQL Server.
 
+
+## Selecting tables
+
+The dm object can be accessed like a named list of tables:
+
+```{r names}
+names(fin_dm)
+fin_dm$loans
+count(fin_dm$trans)
+```
+
+Here we focus on the `loans`, `accounts`, `districts` and `trans` tables:
+
+```{r select}
+fin_dm_small <- fin_dm[c("loans", "accounts", "districts", "trans")]
+```
+
+The `dm_select_tbl()` verb offers a way to do the same in a `%>%` pipe.
```

Comment:
#FIXME example code demonstrating this function

Reply:
Thanks, in af43569.

I'm unsure if introducing the concept that "dm behaves as a list" in passing is a good idea.